packages/testing/playwright/composables/WorkflowComposer.ts

```diff
@@ -22,4 +22,28 @@ export class WorkflowComposer {
 		await responsePromise;
 		await this.n8n.notifications.waitForNotificationAndClose(notificationMessage);
 	}
+
+	async createWorkflow(name?: string) {
```

Comment:
Can we just add a default value for name here?
```suggestion
	async createWorkflow(name = 'My New Workflow') {
		await this.n8n.workflows.clickAddWorkflowButton();
		await this.n8n.canvas.setWorkflowName(name);
		await this.n8n.canvas.saveWorkflow();
	}
```